mkdocs.yml

```
@@ -17,10 +17,12 @@ theme:
       toggle:
         icon: 'material/lightbulb-outline'
         name: 'Switch to light mode'
+  icon:
+    repo: fontawesome/brands/github
 
 repo_name: encode/starlette
 repo_url: https://github.com/encode/starlette
-edit_uri: ""
+edit_uri: https://github.com/encode/starlette/tree/master/docs
```

On other projects I see `edit_uri: blob/master/docs/` instead. Does it makes any difference?

---

This way it forwards to the edition itself!
```suggestion
edit_uri: edit/master/docs/
```